metric/metric.go

```diff
@@ -0,0 +1,58 @@
+package metric
+
+import (
+	"strconv"
+
+	"github.com/prometheus/client_golang/prometheus"
+	log "github.com/sirupsen/logrus"
+)
+
+var logger = log.WithFields(log.Fields{"component": "metrics"})
+
+var (
+	namespace = ""
+	subsystem = "service"
+)
+
+var (
+	reqLabels   = []string{"service", "operation", "code"}
+	reqDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: namespace,
+		Subsystem: subsystem,
+		Name:      "idler_request_duration_seconds",
+		Help:      "Bucketed histogram of processing time (s) of requests.",
+		Buckets:   prometheus.ExponentialBuckets(0.05, 2, 8),
```

Comment:
hope, you have re-evaluated these buckets ranges as per your service.  Also, once you get some metrics recording in production, you should re-check if these buckets are appropriate or not.

Reply:
As you said I am not sure about which are the best buckets, what I have done is just use the same as `wit` and then readjust when we know exactly the values

Of course if you have any advice on how to calculate them in advance(to have some idea) please feel free to share :)